cli/src/main/resources/help/command.txt

```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
+  
+  Arguments should be passed with the commands, in CLI mode command we are having arguments(--help, --help --commands and etc)
+
+  For example, command /subsystem=datasources/data-source.
+  Tab-completion will help completing operation, property names and arguments.
```

Comment:
Do we have completion there for arguments? Is it working? Completion doesn't seem to work for argument value.